File: src/c1removeadjacentduplicates/Java/Main.java
public class Main {
    public static void main(String[] args) {
        String[] inputs = {
                "g",
                "ggaabcdeb",
                "abbddaccaaabcd",
                "aannkwwwkkkwna",
                "abbabccblkklu"
        };

        for (int i = 0; i < inputs.length; i++) {
            String input = inputs[i];
            System.out.println((i + 1) + ".\tRemove duplicates from string: \"" + input + "\"");
            String resultingString = RemoveDuplicatesHelper.removeDuplicates(input);
            System.out.println("\tString after removing duplicates: \"" + resultingString + "\"");
            System.out.println(new String(new char[100]).replace('\0', '-'));
        }
    }
}

File: src/c1removeadjacentduplicates/Python/main.py
from remove_duplicates import remove_duplicates

inputs = [
    "g",
    "ggaabcdeb",
    "abbddaccaaabcd",
    "aannkwwwkkkwna",
    "abbabccblkklu"
]

for i, input_str in enumerate(inputs):
    print(f"{i + 1}.\tRemove duplicates from string: \"{input_str}\"")
    resulting_string = remove_duplicates(input_str)
    print(f"\tString after removing duplicates: \"{resulting_string}\"")
    print("-" * 100)
